src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -669,8 +671,8 @@ public void WindowsSearchPatternInvalid_Wildcards_netcoreapp()
         [PlatformSpecific(TestPlatforms.AnyUnix)]  // Unix-invalid search patterns throws no exception 
         public void UnixSearchPatternInvalid()
         {
-            GetEntries(TestDirectory, "\0");
-            GetEntries(TestDirectory, string.Format("te{0}st", "\0".ToString()));
+            Assert.Throws<ArgumentException>(() => GetEntries(TestDirectory, "\0"));
```

Comment:
This can probably be merged into `WindowsSearchPatternInvalid_Core` and the test can be made cross plat.

Reply:
Oh I see what you mean. Ignore my previous comment. I'll make the Windows test x-platform and will detect the platform internally for the `'/'` case.